frontend/src/metabase/dashboard/components/Dashboard/stickyParameters.js

```diff
@@ -2,6 +2,12 @@ import { isSmallScreen } from "metabase/lib/dom";
 
 export const MAXIMUM_PARAMETERS_FOR_STICKINESS = 5;
 
+const checkIfParametersWidgetShouldBeSticky = dashboard =>
```

Comment:
just moved to the top as functions defined with `const` shouldn't be used before declaration

Reply:
I'd revert the change.

This is only necessary if the function is called directly in the same scope, but `checkIfParametersWidgetShouldBeSticky` is called inside another function and I think this is fine.

Usually I prefer reading from top to bottom, so the previous version where we read `updateParametersWidgetStickiness` first and see it calls `checkIfParametersWidgetShouldBeSticky` inside then we continue reading the implementation below feels more natural than having to jump up in this version.

Reply:
agreed